### ISSUE TITLE
Grantees can see status of file submissions

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/TANF-app#84]

**Comment by amilash** [comment]
@lfrohlich For this epic, is it considered Done with the build we have in place? or is this meant to be feature ready in product? If all subtask functionality is done we can close as 'done' or 'will not do'. Please let me know.

**Comment by amilash** [comment]
@lfrohlich  Let's decide which release we want to target for this work. After speaking with John, he thinks we wont be able to do this work until we have the chron job sending data to old TDP and we are parsing the data already. That tells me this can go for release 2 or 3. We can decide. LMK what you think.

**Comment by ADPennington** [comment]
> @lfrohlich Let's decide which release we want to target for this work. After speaking with John, he thinks we wont be able to do this work until we have the chron job sending data to TDRS and we are parsing the data already. That tells me this can go for release 2 or 3. We can decide. LMK what you think.

**Comment by lfrohlich** [comment]
@amilash  yes, most of this will need to happen after we have parsing/error validation, so am marking as V4. i believe we have sufficient status for tribal MVP, but would be good to briefly discuss

**Comment by ADPennington** [comment]
@sreedevip @reitermb i'm wondering if this ticket needs to be updated based on the latest research findings or closed. is this a dupe of #1269 ? I'm looking for a place to capture brainstorming on how to detect partial submissions.

**Comment by sreedevip** [comment]
@ADPennington It certainly captures all aspects of the Resubmission Epic, but I can also see parts of the Errors epic reflected in the 4th AC

**Comment by sreedevip** [comment]
> > @ADPennington It certainly captures all aspects of the Resubmission Epic, but I can also see parts of the Errors epic reflected in the 4th AC
> 
> @sreedevip could the format of the error guidance be added to #1182? if so, i think this can be closed.

Yes, sounds good @ADPennington

**Comment by ADPennington** [comment]
@valcollignon @lfrohlich assuming the most recent suggestions are reasonable, if #1182 gets updated, this can be closed.

**Comment by lfrohlich** [comment]
@ADPennington  what about all of the issues in the backlog that are still open tied to this epic? should those be in a separate epic?

**Comment by ADPennington** [comment]
> @ADPennington what about all of the issues in the backlog that are still open tied to this epic? should those be in a separate epic?

@lfrohlich i think this is a scenario where #84, and #1269 can be merged or we need to distinguish them better.

**Comment by valcollignon** [comment]
> @lfrohlich i think this is a scenario where #84, and #1269 can be merged or we need to distinguish them better.

I'm not sure we should merge #84 and #1269, since #84 is not UX/UI specific and #1269 is. If this epic gets closed, I think the issues tied to it will not have an epic. I would rather move issues out of this epic into a new/existing epic first. This epic has been open for a long time.

**Comment by reitermb** [comment]
> > @lfrohlich i think this is a scenario where #84, and #1269 can be merged or we need to distinguish them better.
> 
> I'm not sure we should merge #84 and #1269, since #84 is not UX/UI specific and #1269 is. If this epic gets closed, I think the issues tied to it will not have an epic. I would rather move issues out of this epic into a new/existing epic first. This epic has been open for a long time.

No objections from UX if we wanted to repurpose 1269 as a cross-functional epic

**Comment by sreedevip** [comment]
> > > @lfrohlich i think this is a scenario where #84, and #1269 can be merged or we need to distinguish them better.
> > 
> > 
> > I'm not sure we should merge #84 and #1269, since #84 is not UX/UI specific and #1269 is. If this epic gets closed, I think the issues tied to it will not have an epic. I would rather move issues out of this epic into a new/existing epic first. This epic has been open for a long time.
> 
> No objections from UX if we wanted to repurpose 1269 as a cross-functional epic

@valcollignon @ADPennington @valcollignon @reitermb the way I see it, [#84](https://github.com/raft-tech/TANF-app/issues/84) has both elements of Resubmission Epic [1269] (https://github.com/raft-tech/TANF-app/issues/1269) and Errors Epic [1182] (https://github.com/raft-tech/TANF-app/issues/1182) in it. All the supporting tickets for this epic [#84](https://github.com/raft-tech/TANF-app/issues/84) have already been covered in the Resubmission Epic 1269] (https://github.com/raft-tech/TANF-app/issues/1269)  and through the User Access Release 1 [#1183] (https://github.com/raft-tech/TANF-app/issues/1183). And those supporting tickets( [125] (https://github.com/raft-tech/TANF-app/issues/125), [126] (https://github.com/raft-tech/TANF-app/issues/126), [127] (https://github.com/raft-tech/TANF-app/issues/127), [128] (https://github.com/raft-tech/TANF-app/issues/128), [130] (https://github.com/raft-tech/TANF-app/issues/130), [1216] (https://github.com/raft-tech/TANF-app/issues/1216)) are not well-refined either. My recommendation would be to close [#84](https://github.com/raft-tech/TANF-app/issues/84) after merging what's relevant to other tickets/ epics

**Comment by valcollignon** [comment]
Let's make sure to discuss this during backlog refinement on 11.9.
cc: @lfrohlich @ADPennington @sreedevip @reitermb

**Comment by valcollignon** [comment]
Impact session will include refining these tickets. We will use this epic for dev tickets and this should not be closed out, as was alluded to in previous comments. Per backlog refinement 1.11.22.
CC: @abottoms-coder @lfrohlich @reitermb
